NGitLab/Models/VersionedDiffResult.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Text.Json.Serialization;
+
+namespace NGitLab.Models;
+
+public class VersionedDiffResult
+{
+    [JsonPropertyName("id")]
+    public long Id { get; set; }
+
+    [JsonPropertyName("base_commit_sha")]
+    public long BaseCommitSha { get; set; }
```

Comment:
According to the doc, this should be a `string`. However, since it represents a SHA, it could also be of type `Sha1`.